backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -67,4 +67,13 @@ export class TransactionLinkResolver {
 
     return new TransactionLink(transactionLink, new User(user))
   }
+
+  @Authorized([RIGHTS.QUERY_TRANSACTION_LINK])
+  @Query(() => TransactionLink)
+  async queryTransactionLink(@Arg('code') code: string): Promise<TransactionLink> {
+    const transactionLink = await dbTransactionLink.findOneOrFail({ code })
+    const userRepository = getCustomRepository(UserRepository)
+    const user = await userRepository.findOneOrFail({ id: transactionLink.userId })
+    return new TransactionLink(transactionLink, new User(user))
```

Comment:
What happens when `user = null`?

Reply:
Fail so it will throw an exception on line 76